backend/src/schema/resolvers/helpers/existingEmailAddress.js

```diff
@@ -0,0 +1,26 @@
+import { UserInputError } from 'apollo-server'
+export default async function alreadyExistingMail(_parent, args, context) {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-02T09:39:05Z" title="Wednesday, October 2nd 2019, 11:39:05 am +02:00">Oct 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,26 @@
+import { UserInputError } from 'apollo-server'
+export default async function alreadyExistingMail(_parent, args, context) {
```


was this name `alreadyExistingMail` on purpose? 
Shouldn't it be `alreadyExistingEmail`?
or even the same as the file name? `existingEmailAddress`??

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-02T13:30:44Z" title="Wednesday, October 2nd 2019, 3:30:44 pm +02:00">Oct 2, 2019</time>_

---


Not on purpose. Next time I touch this file I will go with `existingEmailAddress`. This makes sense especially because we have multiple nodes for similar things now.